erdos/src/dataflow/stream/mod.rs

```diff
@@ -55,37 +59,48 @@ pub trait WriteStreamT<D: Data> {
 }
 
 pub trait Stream<D: Data> {
-    fn name(&self) -> String {
-        default_graph::get_stream_name(&self.id())
-    }
-    fn set_name(&mut self, name: &str) {
-        default_graph::set_stream_name(&self.id(), name);
-    }
+    fn name(&self) -> String;
     fn id(&self) -> StreamId;
+    fn get_graph(&self) -> Arc<Mutex<InternalGraph>>;
```

Comment:
```suggestion
    fn graph(&self) -> Arc<Mutex<InternalGraph>>;
```

Since everything seems to be following the convention of not having `get_` in the prefix, can you change this too?